src/include/cell_inline.h

```diff
@@ -705,17 +697,18 @@ __wt_cell_unpack_safe(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CE
     copy.len = 0; /* [-Wconditional-uninitialized] */
     copy.v = 0;   /* [-Wconditional-uninitialized] */
 
-    ps = unpack_addr == NULL ? NULL : &unpack_addr->ps;
-
     if (unpack_addr == NULL) {
         unpack = (WT_CELL_UNPACK_COMMON *)unpack_value;
+        ps = &unpack_value->ovfl_ps;
```

Comment:
We should not do anything to unpack the page stats in this function.

Reply:
Removed the page stat unpacking from the cell unpack flow here: [7fdd194](https://github.com/wiredtiger/wiredtiger/pull/8792/commits/7fdd194a5742efa7716451b29b381812ed336f2c)